plugin/reload/README.md

```diff
@@ -2,7 +2,9 @@
 
 ## Name
 
-*reload* - allows automatic reload of a changed Corefile
+*reload* - allows automatic reload of a changed _Corefile_<sup>*</sup>
+
+<sub>* To enable automatic reloading of a _zone file_, use the [`auto`](https://github.com/coredns/coredns/blob/master/plugin/auto/README.md) plugin.</sub>
```

Comment:
Good to have, but this may (haven't check the sync script for getting it on coredns.io). Also `<sub>` def. break the conversion into man pages.

Good to have this warning but it probably needs to move into description, or See Also (but that may be too far down?)

Reply:
I've removed the links and sub tags and moved the text to the Description section.